docs/en/sql-reference/functions/conditional-functions.md

```diff
@@ -1,16 +1,16 @@
 ---
-description: 'Documentation for Conditional Functions'
+description: 'Documentation for conditional functions'
 sidebar_label: 'Conditional'
 sidebar_position: 40
 slug: /sql-reference/functions/conditional-functions
-title: 'Conditional Functions'
+title: 'Conditional functions'
 ---
 
-# Conditional Functions
+# Conditional functions
 
 ## Overview {#overview}
 
-### Using Conditional Results Directly {#using-conditional-results-directly}
+### Using Conditional results directly {#using-conditional-results-directly}
```

Comment:
Okay, more lowercasing here ... I am good with that, but perhaps we do one big PR which applies that spelling _everywhere_?